pytradfri/device.py

```diff
@@ -202,10 +203,22 @@ def set_dimmer(self, dimmer, *, index=0, transition_time=None):
 
         return self.set_values(values, index=index)
 
+    def set_color_temp(self, color_temp, *, index=0):
+        """Set color temp a light."""
+        if self.can_set_mireds:
+            white_spectrum_bulb = True
+        else:
+            white_spectrum_bulb = False
+
+        values = kelvin_to_xyY(color_temp, white_spectrum_bulb)
```

Comment:
Rather than accepting kelvin, can we just accept mireds instead?

Reply:
@lwis thanks! Got home a bit to late this evening but will update the code tomorrow.